config/urls.py

```diff
@@ -16,9 +16,14 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.conf.urls.static import static
+from config import settings
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('general/', include('apps.general.urls')),
-    path('master/', include('apps.master.urls')),
+    path('api/v1/customer/', include('apps.customer.urls')),
+    path('api/v1/masters/', include('apps.masters.urls')),
 ]
```

Comment:
Is end "/" is needed here api/v1/customer/ ?

Reply:
Yes Thulasi. 
if we were not given like that, it will take path: api/v1/customercustomer/ like this way.